controls/V-233562.rb

```diff
@@ -101,6 +101,10 @@
 		 describe "The #{pg_audit_log_dir} directory was not found. Check path for this postgres version/install to define the value for the 'pg_audit_log_dir' inspec input parameter." do
 		   skip "The #{pg_audit_log_dir} directory was not found. Check path for this postgres version/install to define the value for the 'pg_audit_log_dir' inspec input parameter."
 		 end 
+
+		 describe sql.query('SHOW shared_preload_libraries;', [pg_db]) do
+			its('output') { should include 'pgaudit' }
+		  end

```

Comment:
@damif5 Thanks for adding this check! It's currently placed under the `else` block but it should run regardless of any condition so the describe block can be moved before the `if` statement. I also noticed that one line was missing which is a pre-requisite so please add this before the describe block:
```
sql = postgres_session(pg_dba, pg_dba_password, pg_host, input('pg_port'))
```

This will allow `sql.query` in that describe block to run that query after establishing a connection :)

Reply:
completed as advised.